Scripts/Services/LootGeneration/RunicReforging/RunicReforging.cs

```diff
@@ -941,30 +941,12 @@ public static bool ApplyResistance(Item item, int value, AosElementAttribute att
 
         public static int Scale(int min, int max, int perclow, int perchigh, int luckchance, bool reforged)
         {
-            int percent;
-
-            if (reforged)
-            {
-                percent = Utility.RandomMinMax(perclow, perchigh);
-            }
-            else
-            {
-                percent = Utility.RandomMinMax(0, perchigh);
-            }
-
+            decimal percent = Utility.RandomMinMax(perclow, perchigh);
```

Comment:
A minimum value is always calculated on loot, but was ignored. So the low is now being used again here, to stop everything rolling from 0 upwards